mteb/tasks/Classification/ita/ItaHateClassification.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class ItaHateClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="ItaHateClassification",
+        dataset={
+            "path": "Paul/hatecheck-italian",
+            "revision": "21e3d5c827cb60619a89988b24979850a7af85a5",
+        },
+        description="""Hate speech detection dataset with binary
+                       (hateful vs non-hateful) labels. Includes 25+ distinct types of hate
+                       and challenging non-hate. Multilingual datase released as 10 unilingual models
+                     """,
+        reference="https://aclanthology.org/2022.woah-1.15/",
```

Comment:
Seems like this is a multilingual dataset. Just letting you know that it is possible to add the full dataset using a multilingual task. This would also allow us to cover Dutch and Portuguese.